Symfony/CS/Tests/Fixer/Symfony/OperatorsSpacesFixerTest.php

```diff
@@ -30,6 +31,72 @@ public function provideCases()
     {
         return array(
             array(
+                '<?php $a++;',
+                '<?php $a ++;',
+            ),
+            array(
+                '<?php $a--;',
+                '<?php $a --;',
+            ),
+            array(
+                '<?php ++$a;',
+                '<?php ++ $a;',
+            ),
+            array(
+                '<?php --$a;',
+                '<?php -- $a;',
+            ),
+            array(
+                '<?php $a = !$b;',
+                '<?php $a = ! $b;',
+            ),
+            array(
+                '<?php func(+$a, -2,-$b);',
+                '<?php func(+ $a, - 2,- $b);',
+            ),
+            array(
+                '<?php $a = &$b;',
+                '<?php $a = & $b;',
```

Comment:
IMO this should be two fixers.
One for binary ops., one for unary ops.
I prefer `&` sticked to variable, but not all do.